server/src/domain/media/media.util.ts

```diff
@@ -555,3 +562,59 @@ export class VAAPIConfig extends BaseHWConfig {
     return this.config.cqMode !== CQMode.ICQ || this.config.targetVideoCodec === VideoCodec.VP9;
   }
 }
+
+export class RKMPPConfig extends BaseHWConfig {
+  getOptions(videoStream: VideoStreamInfo, audioStream?: AudioStreamInfo | undefined): TranscodeOptions {
+    const options = super.getOptions(videoStream, audioStream);
+    options.ffmpegPath = 'ffmpeg_mpp';
+    options.ldLibraryPath = '/lib/aarch64-linux-gnu:/lib/ffmpeg-mpp';
+    options.outputOptions.push(...this.getSizeOptions(videoStream));
+    return options;
```

Comment:
Up to preference, but I think this syntax is more idiomatic:

```
{
   ...options,
   ffmpegPath: 'ffmpeg_mpp',
   ldLibraryPath = '/lib/aarch64-linux-gnu:/lib/ffmpeg-mpp'
}
```

Reply:
tried but it's way longer/complex with with outputOptions. so I left it as it is.